packages/components/scripts/wrapper-generator/AbstractWrapperGenerator.ts

```diff
@@ -76,8 +78,15 @@ export abstract class AbstractWrapperGenerator {
     const componentExports = this.relevantComponentTagNames
       .map((component) => {
         const componentSubDir = this.getComponentSubDir(component);
-        const componentFileName = this.getComponentFileName(component, true);
-        return `export * from './${componentSubDir ? componentSubDir + '/' : ''}${componentFileName}';`;
+        const componentFileNameWithoutExtension = path.parse(this.getComponentFileName(component)).name;
+        const getBarrelFileContent = this.getBarrelFileContent(
+          componentFileNameWithoutExtension,
+          componentSubDir
+        );
+
+        return getBarrelFileContent
```

Comment:
Ternary could be replaced by `return getBarrelFileContent || whatever` but this is obsolete after the changes of the previous comment are integrated.